### PR TITLE
Improved readme and website.

### PR DESCRIPTION
**Pull request by zrzjohn**
> I had a problem with my repository before. I fork again.

### Previous Operations

> Create English & CHT Website: 添加了网站的英文页面和繁体中文页面，并改善了您的网站页面（具体请您浏览 zrzjohn / LGBTQIA-in-China ）。
> 
> Improved Readme: 添加了英文和繁体中文的Readme，并改善了语言选择菜单。

### New Operation
- Move signature to SIGN.md .
<https://github.com/zrzjohn/LGBTQIA-in-China/blob/master/SIGN.md>

- Create MIT License.
<https://github.com/zrzjohn/LGBTQIA-in-China/blob/master/LICENSE>

### CODE REVIEW

File: SIGN.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Requirement
```
Comment (KevinZonda) [inline SIGN.md:1]: **KEEP SIGNATURE IN README.MD UNLESS YOU HAVE WAY TO SHOW IT IN MAIN PAGE!**

File: index.html [hunk 2]
```diff
@@ -362,13 +364,9 @@ <h2>如何尊重 LGBTQIA 群体？</h2>
...
 				<h1>支持</h1>
 				<hr>
-				<p>如果你支持 LGBTQIA 平权，欢迎在本项目 GitHub 仓库的 README.md 中留下你的名字，并点一个 star 作为支持！同时，你也可以写一篇关于（中国）LGBTQIA
-					群体的文章，将原文地址和 md 文件（作为备份）提供给我们，只要文章内容与 LGBTQIA 相关，或是 LGBTQIA 中的任意一个群体相关，呼吁平权，我们都会通过。</p>
-				<p style="color:gray"><i class="iconfont icon-fork"></i> 尽量以 Pull Requests 的方式提交 commit（文章请提交在 post 目录下）
-				</p>
-				<p>所有支持者的名字和他们提供的文章都可以在 GitHub 仓库下的 <a
-						href="https://github.com/LGBT-CN/LGBTQIA-in-China/blob/master/README.md">README.md</a> 文件中看到。
-				</p>
+				<p>如果你支持 LGBTQIA 平权，欢迎在本项目 <a href="https://github.com/LGBT-CN/LGBTQIA-in-China/">GitHub 仓库</a> 的 SIGN.md 中留下你的名字，并点一个 Star 作为支持！同时，你也可以写一篇关于中国 LGBTQIA
```
Comment (KevinZonda) [inline index.html:367]: No need.

File: traditional-chinese.html [hunk 3]
```diff
@@ -0,0 +1,409 @@
...
+	<title>LGBTQIA in China</title>
+	<link
+		href="https://fonts.googleapis.com/css?family=Noto+Sans+SC:300|Noto+Serif+SC:300|ZCOOL+KuaiLe|ZCOOL+XiaoWei&display=swap"
+		rel="stylesheet">
+	<link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/twbs/bootstrap/dist/css/bootstrap-grid.min.css">
+	<link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/alvarotrigo/fullPage.js/dist/fullpage.min.css">
+	<link rel="stylesheet" href="https://at.alicdn.com/t/font_1640253_7ac8t2q6btg.css">
+	<link rel="stylesheet" href="https://at.alicdn.com/t/font_1912611_olza2ei6vv.css">
+	<style>
+		body {
```
Comment (KevinZonda) [inline traditional-chinese.html:24]: Detach `<style>` part to css file.

**Comment by KevinZonda** [comment]
@scvoet @zrzjohn I don't think it's neccessary to detach signature part to a single file.

**Comment by KevinZonda** [comment]
> So, what open-source license do you choose?
> 
> It is recommended that you refer to https://choosealicense.com

It's under discussion.

**Comment by KevinZonda** [comment]
@Borschts Hi there, plz review CHT translate. Thanks!

**Comment by zrzjohn** [comment]
> 
> 
> @scvoet @zrzjohn I don't think it's neccessary to detach signature part to a single file.

I don't recommend it. Because neither 996.ICU nor wuhan2020 put contributors in readme.

If you really want to, please sign.md Replace with readme.md (in HTML file) ([link](https://preview.yozodcs.com/fcscloud/view/preview/Z84UMQNG3cnYSfkd_XeffJs5SjiBEdsmAFhaBSB90VUSn-VPS1RTlHAN7QguUSKftwKApJGcYhUnkd3hl3AzyjhEFhmkuNRyvsCMZ2bK26isMWc4q22HHeMB4pxdaSrOjKKNLvAaWhBx8DRHqkHQVyitTzmYVnUtfyn4CuYWE8kgnfcr4oskqpW1FhIq7iB6EBkyc6V6sh5V0W7qC_Lwztkshhi7LSd63cg1Nt2T74yXmToND27kRscx4VnWLVxn_zSIFNCpvVceRvMcqQnXhY9UM-jh9JQli-xhab7FLXgbckdLZJ2D5tOYGJCrNo9Ng7gHo7H0L75slz9Egek76rtXr_yGGHk4/QQ%E5%9B%BE%E7%89%8720200813085614.files/file0001.png)).

Warning SIGN.md replace to README.md .

**Comment by zrzjohn** [comment]
@KevinZonda 

I think that Chinese people can be in China. README.md Signature, foreigners can sign in README.EN.md Taiwanese can sign in the README.ZH.md Signed in.

File: index.html (hunk 2 above)
Comment (KevinZonda) [reply index.html]: Solved in 563c3b3

File: traditional-chinese.html (hunk 3 above)
Comment (KevinZonda) [reply traditional-chinese.html]: Solved in 563c3b3

File: README.md [hunk 4]
```diff
@@ -6,48 +6,28 @@
...
 本仓库意图在 LGBTQIA 程序员群体中唤起对自由平等的追求，哪怕我们都深知这不会激起浪花。
 
 你可以访问[这个网站](https://cnlgbt.org)查看更多信息。
 
-## 参与
+## 如何参与
 
-1. 在 README.md 中署名 ([署名](#署名))
-2. 写篇关于中国 LGBTQIA 群体的文章（[文章](#文章)）
+1. 在 SIGN.md 中[署名](README.md#署名)；
```
[the remark this answers is not in the document]
Comment (KevinZonda) [reply README.md]: Solved in 563c3b3

File: SIGN.md (hunk 1 above)
Comment (KevinZonda) [reply SIGN.md]: Solved in 563c3b3

File: README.ZH.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+1. 在 SIGN.md 中[署名](README.ZH.md#署名)；
+2. 寫篇關於中國 LGBTQIA 群體的[文章](README.ZH.md#文章)。
+
+## 署名
+
+> 這將造成壹定程度上的隱私泄漏，因為在中國坦白自己的性取向、性別認同及性別表達仍是個風險較大的行為。
+
+您可以在[SIGN.md](SIGN.md#Supporters)中進行署名或查看署名者。
+
+## 文章
```
[the remark this answers is not in the document]
Comment (KevinZonda) [reply README.ZH.md]: Solved in 563c3b3

File: README.EN.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+1. [Sign](README.EN.md#Signature) in SIGN.md;
+2. Write an [article](README.EN.md#Articles) about Chinese LGBTQIA community.
+
+## Signature
+
+> This may create a degree of privacy leakage, as confessing one's sexual orientation, gender identity and gender expression remains a risky activity in China.
+
+You can view supporters list in [SIGN.md](SIGN.md#Supporters).
+
+## Articles
```
[the remark this answers is not in the document]
Comment (KevinZonda) [reply README.EN.md:26]: Solved in 563c3b3

**Comment by KevinZonda** [comment]
@scvoet Hi there! Plz review!

**Comment by KevinZonda** [comment]
> @KevinZonda I think that Chinese mainland people can signature in README.md , foreigners can signature in README.EN.md, Taiwanese can signature in the README.ZH.md .

Exactly, this is far beyond what we should maintain unless we have a bot to help us sync SIGN.md to README*.md.

**Comment by zrzjohn** [comment]
> 
> 
> > @KevinZonda I think that Chinese mainland people can signature in README.md , foreigners can signature in README.EN.md, Taiwanese can signature in the README.ZH.md .
> 
> Exactly, this is far beyond what we should maintain unless we have a bot to help us sync SIGN.md to README*.md.

Can consider develop a robot to help us sync.

**Pull request merged**